reana/reana_dev/utils.py

```diff
@@ -790,3 +790,24 @@ def wrapper(*args, **kwargs):
         return func(*args, **kwargs)
 
     return wrapper
+
+
+def validate_directory(ctx, param, target_directory):
+    """Validate if directory is valid for first time cloning."""
+    if target_directory:
+        instalation_directory = os.path.realpath(target_directory)
```

Comment:
Should be `installation_directory` instead (double L).

What about just using the same variable to assign the real directory?

``` python
target_directory = os.path.realpath(target_directory)
```

I see that in the end, we're returning `target_directory` instead of `installation_directory`. Wouldn't be better to return the real path? Then we wouldn't need [this assignment](https://github.com/reanahub/reana/pull/523/files#diff-9953f1f34f7afe5aec70e06c77e2a69473ce7c4cb35bdbb20b3c0985dd858c91R383).